jaseci_core/jaseci/api/config_api.py

```diff
@@ -47,25 +52,61 @@ def config_set(self, name: str, value: str or dict, do_check: bool = True):
         self._h.save_glob(name, value)
         return [f"Config of '{name}' to '{value}' set!"]
 
+    @Interface.admin_api()
+    def config_yaml(self, name: str, file: list):
```

Comment:
If I am understanding this correctly, this is basically you can use yaml file to update the config. What is the problem you are trying to address here vs. the existing config set API?

Also, for the `file` parameter, does the yaml file need to be already on the filesystem of the jaseci instance?